src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const robotReverse = {};
```

Comment:
![image](https://user-images.githubusercontent.com/35385361/101774883-61f96700-3af7-11eb-8fbd-be348fa7c672.png)
```suggestion
  const reversedRobot = {};
```